### DISCUSSION TITLE
Congratulations!

### DISCUSSION THREAD

**Discussion by Kitick** [flandreas/antares#482] [General]
Hello, its me again, I purely wanted to say congrats on getting Antares to 1.0! I personally love the work you've done so far with this program, its becoming more and more of a valuable tool when I'm doing circuits and I personally find it more ui friendly and useful than logicism at this point, at least for what I'm using it for. Keep up the amazing work, can't wait to see what the future holds! And heck if you ever open up donations I would love to support its development further!

**Comment by flandreas** [comment]
@Kitick Thank you, much appreciated!
